parachain/runtime/snowbase/src/lib.rs

```diff
@@ -639,7 +641,7 @@ impl ethereum_beacon_client::Config for Runtime {
 	type MaxFinalizedHeaderSlotArray = MaxFinalizedHeaderSlotArray;
 	type ForkVersions = ChainForkVersions;
 	type WeakSubjectivityPeriodSeconds = WeakSubjectivityPeriodSeconds;
-	type WeightInfo = weights::ethereum_beacon_client::SnowbridgeWeight<Self>;
+	type WeightInfo = ethereum_beacon_client::weights::SnowbridgeWeight<Self>;
```

Comment:
The weights used here should be snowbase/src/weights/ethereum-beacon-client (which looks like it is being deleted). Looks like this changes the weights to use the pallet weights. Why this change?

Reply:
Yes, but seems we're still missing benchmarked weights for runtime [snowblink](https://github.com/Snowfork/snowbridge/blob/c3bd614dffad1dae5996b8b67e6cfd3023669111/parachain/runtime/snowblink/src/lib.rs#L640) and [snowbridge](https://github.com/Snowfork/snowbridge/blob/c3bd614dffad1dae5996b8b67e6cfd3023669111/parachain/runtime/snowbridge/src/lib.rs#L640) which I would assume more important than `snowbase` the test runtime. 

And also as benchmark required, we need to do it in a specific hardware same as deploy in production which I'm not sure finalized yet.

So I would prefer we make things easy as now and rebenchmark it in a more strict and consist manner.

Reply:
Sounds good to me.

Reply:
Makes sense, thank you!